content/en/blog/_posts/2021-11-02-csi-migration-status.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: blog
+title: "Kubernetes 1.23: Kubernetes In-Tree to CSI Volume Migration Status Update"
+date: 2021-11-03T15:00:00+08:00
+slug: kubernetes-1-23-feature-csi-migration-status-update
```

Comment:
I think the slug usually does not contain the release version such as "kubernetes-1-23".